lib/fetch.ts

```diff
@@ -0,0 +1,42 @@
+async function fetchHelper(path: string, params: { [key: string]: string }) {
+    let headers = new Headers();
+    if (process.env.INSTANCE === 'staging') {
+        const auth =
+            'Basic ' +
+            Buffer.from(
+                `${process.env.AUTH_USER}:${process.env.AUTH_PASSWORD}`,
+            ).toString('base64');
+        headers.append('Authorization', auth);
+    }
+    return await fetch(
+        `${process.env.BASE_URL}api/v2/pages/${path}?` +
+            new URLSearchParams(params),
+        { headers: headers },
+    );
+}
+
+async function getAreWeHeadlessYetHomePageID() {
+    const response = await fetchHelper('', {
+        type: 'areweheadlessyet.AreWeHeadlessYetHomePage',
+    });
+    const result = await response.json();
+    return result.items[0].id;
+}
+
+const camelize = require('camelize');
```

Comment:
I'm sorry, this is my bad - I recommended camelize before, but I've since checked another project and we were using `humps` - https://www.npmjs.com/package/humps?activeTab=readme - It doesn't make much difference but this package seems more popular and better maintained. This also avoids the `require` here and we can import it normally at the top of the file.

```
import humps from 'humps';

...

return humps.camelizeKeys(response);
```